test/model/dataTypes/MemberVariable.test.ts

```diff
@@ -0,0 +1,186 @@
+import "jest-extended";
+import Element from "../../../src/model/modelInfo/Element";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing file extension for "../../../src/model/modelInfo/Element"](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5691315)